geopandas/geodataframe.py

```diff
@@ -2202,6 +2203,10 @@ def sjoin_nearest(
         distance_col : string, default None
             If set, save the distances computed between matching geometries under a
             column of this name in the joined GeoDataFrame.
+        exclusive : bool, optional, default False
+            If True, the nearest geometries that are equal to the input geometry
+            will not be returned, default False
```

Comment:
```suggestion
            will not be returned, default False.
```

(in the online docs, the next line will continue here, so it will look strange without punctuation)